tool/tctl/common/plugin/awsic.go

```diff
@@ -316,3 +317,58 @@ func (p *PluginsCommand) InstallAWSIC(ctx context.Context, args installPluginArg
 
 	return nil
 }
+
+type awsICUpdateCredsArgs struct {
+	cmd        *kingpin.CmdClause
+	pluginName string
+	payload    string
+}
+
+func (p *PluginsCommand) initUpdateCredsAWSIC(parent *kingpin.CmdClause) {
+	p.updateCreds.awsic.cmd = parent.Command("awsic", "Update the AWS Identity Center SCIM bearer token")
+	cmd := p.updateCreds.awsic.cmd
+
+	cmd.Flag("plugin-name", "Name of the AWSIC plugin instance to update. Defaults to "+apicommon.OriginAWSIdentityCenter+".").
+		Default(apicommon.OriginAWSIdentityCenter).
+		StringVar(&p.updateCreds.awsic.pluginName)
+
+	cmd.Arg("token", "The new SCIM bearer token.").
+		PlaceHolder("TOKEN").
+		Required().
+		StringVar(&p.updateCreds.awsic.payload)
+}
+
+func (p *PluginsCommand) UpdateAWSICCreds(ctx context.Context, args installPluginArgs) error {
+	plugin, err := args.plugins.GetPlugin(ctx, &pluginsv1.GetPluginRequest{
+		Name:        p.updateCreds.awsic.pluginName,
+		WithSecrets: true,
+	})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
```

Comment:
Do we need to make sure that the `plugin` that was returned is an AWSIC plugin? What happens if you run this command and pass it the name of some other plugin?